sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -150,6 +152,7 @@ class DetectLanguageResult(DictMixin):
     def __init__(self, **kwargs):
         self.id = kwargs.get("id", None)
         self.detected_languages = kwargs.get("detected_languages", None)
+        self.primary_language = kwargs.get("primary_language", None)
```

Comment:
curious: What happens if  primary language is None?

Reply:
Sameeksha asked this above: It's listed as required in the swagger, so it won't ever be None. If it can't detect the language it will still return an object like {'name': '(Unknown)', 'iso6391_name': '(Unknown)', 'score': 0.0}